pybandits/base.py

```diff
@@ -129,9 +129,9 @@ def _get_valid_actions(self, forbidden_actions: Optional[Set[ActionId]]) -> Set[
         return valid_actions
 
     def _check_update_params(
-        self,
-        actions: List[ActionId],
-        rewards: List[Union[NonNegativeInt, List[NonNegativeInt]]],
+            self,
```

Comment:
I guess this indentation is a mistake, right?

Reply:
I guess it's some auto-format thing. 
Reverting